suse_ha-formula/suse_ha/macros.jinja

```diff
@@ -68,11 +68,22 @@ ha_resource_start_{{ resource }}:
        - cmd: ha_resource_update_{{ resource }}
 {%- endmacro %}
 
-{%- macro property(option, value) -%}
+{%- macro property(option, value=None) -%}
+{%- if value is none %}
+{%- set value = management.get(option) %}
+{%- if value is none %}
+{%- set value = management.get(option.replace('-', '_')) %}
+{%- endif %}
+{%- endif %}
```

Comment:
You can probably (untested) do this with one if condition:
```
{%- if value is none %}
{%- set value = management.get(option, management.get(option.replace('-', '_'))) %}
{%- endif %}
```

Reply:
Thanks for the idea! Seems to work.